.circleci/config.yml

```diff
@@ -21,15 +21,16 @@ jobs:
       - run:
           name: Building and packaging
           command: |
-            /home/circleci/tooling/build-tools/build_tools.sh composer -o "install --no-dev"
-            rm -rf vendor/phpoffice/phpspreadsheet/{bin,docs,samples}
+            /home/circleci/tooling/build-tools/build_tools.sh composer -o "build"
+            rm -rf src
             if [ "${CIRCLE_BRANCH}" != "main" ]; then
               /home/circleci/tooling/build-tools/build_tools.sh package_build -o "gf-entries-in-excel gfexcel.php --include-hash"
             else
               /home/circleci/tooling/build-tools/build_tools.sh package_build -o "gf-entries-in-excel gfexcel.php $([[ $(git log -n 1 | grep "\[skip release\]") ]] && echo --include-hash)"
             fi
             mkdir -p gf-entries-in-excel
-            mv vendor gf-entries-in-excel
+            rm -rf build/composer.* build/strauss.phar
```

Comment:
@doekenorg is this necessary if we have https://github.com/GravityKit/GravityExport-Lite/pull/181/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R11?